lms/product/plugin/grouping_service.py

```diff
@@ -24,6 +24,21 @@ class GroupingServicePlugin:  # pragma: nocover
     sections_type: Grouping.Type = None
     """The type of sections this plugin supports. `None` disables support."""
 
+    deep_linking = False
+    """
+    Does this product use deep linking to configure assignments?
+
+    If so we will read group set info from the URL rather than the DB.
+    """
+
+    def get_grouping_type(self, svc) -> Grouping.Type:
+        """Get the type of grouping used in this launch."""
+
+        if svc.get_group_set_id():
+            return Grouping.Type.GROUP
+
```

Comment:
I'll add a `if self.sections_enabled: return SECTIONS`

And move most of the current overriden `get_grouping_type` to that sections_enabled plugin (and have the default be `return False`